infra/helper.py

```diff
@@ -205,6 +224,8 @@ def build_fuzzers(build_args):
 def run_fuzzer(run_args):
   """Runs a fuzzer in the container."""
   parser = argparse.ArgumentParser('helper.py run_fuzzer')
+  _add_engine_args(parser)
```

Comment:
Yes sounds good, yes i think we write the fuzzing config in some file in /out and then both run_fuzzer and reproduce use it.